tc/sgx/trusted_worker_manager/enclave/kme_workload/kme_workload_plug-in.h

```diff
@@ -50,18 +50,15 @@ class KMEWorkloadProcessor: public WorkloadProcessor {
 
     void GetUniqueId(
         const std::vector<tcf::WorkOrderData>& in_work_order_data,
-        std::vector<tcf::WorkOrderData>& out_work_order_data,
-	ExtWorkOrderInfoKME* ext_wo_info_kme);
+        std::vector<tcf::WorkOrderData>& out_work_order_data);
 
     void Register(
         const std::vector<tcf::WorkOrderData>& in_work_order_data,
-        std::vector<tcf::WorkOrderData>& out_work_order_data,
-	ExtWorkOrderInfoKME* ext_wo_info_kme);
+        std::vector<tcf::WorkOrderData>& out_work_order_data);
 
     void PreprocessWorkorder(
         const std::vector<tcf::WorkOrderData>& in_work_order_data,
-        std::vector<tcf::WorkOrderData>& out_work_order_data,
-	ExtWorkOrderInfoKME* ext_wo_info_kme);
+        std::vector<tcf::WorkOrderData>& out_work_order_data);
 
     void ProcessWorkOrder(
```

Comment:
Can you change method name to something else. We already have a same method name in WorkOrderProcessor class as well.

Reply:
Its overriden function from the same interface.

Reply:
Could you add an `override` keyword in header file. for override functions in derived class That will serve two purpose 1) Accidentally changing the function signature of overriding function , 2)  Tell reader that this is an overridden function

Reply:
Added override keyword.